src/coreclr/jit/gentree.h

```diff
@@ -1761,6 +1761,9 @@ struct GenTree
     inline bool IsVectorAllBitsSet() const;
     inline bool IsVectorBroadcast(var_types simdBaseType) const;
     inline bool IsMaskAllBitsSet() const;
+    inline bool IsMaskAllBitsZero() const;
```

Comment:
nit: We just use the term `Zero` elsewhere, such as `IsVectorZero`, so this should probably also be `IsMaskZero` for parity.

-- We use the name based on the managed API, for the most part. So we're basically checking, is this `AllBitsSet` or is this `Zero`